examples/developer-tools/src/serialization.ts

```diff
@@ -59,6 +59,29 @@ export function loadBlock(workspace: Blockly.Workspace, blockName?: string) {
   }
 }
 
+/**
+ * Loads given block json into the given workspace.
+ *
+ * @param workspace Blockly workspace to load into.
+ * @param blockJson Block json to load.
+ */
+export function loadBlockFromData(
+  workspace: Blockly.Workspace,
+  blockJson: Blockly.serialization.blocks.State,
+) {
+  // Disable events so we don't save while deleting blocks.
+  Blockly.Events.disable();
+  workspace.clear();
+  Blockly.Events.enable();
+
+  // There might be conflicts when loading a block from a file.
+  // If so, find a similar unused name so we don't overwrite.
+  const blockName = getNewUnusedName(blockJson.fields.NAME);
+  blockJson.fields.NAME = blockName;
+
+  Blockly.serialization.blocks.append(blockJson, workspace);
```

Comment:
I think you can change this to just call `load` instead of `clear` and `append`? Unless `getNewUnusedName` does something I don't know about.

Reply:
Discussed offline since this is just one block data that no. I updated the tsdoc for the function to be more explicit it's just one block's data, lmk if that helps